website/frontend/templates/docs/plugin-api.html

```diff
@@ -71,6 +75,37 @@ <h1 id="file_formats">File Formats</h1>
 
       register_format(MyFile)</pre>
 
+	  <p>
+		Variables explanation:
+		<ul>
+			<dl>"PLUGIN_DESCRIPTION" should be as simple as possible, while still describing the main function.</dl>
+			<dl>"PLUGIN_VERSION" should be filled with the version of Plugin.</dl>
+			<dl>"PLUGIN_API_VERSIONS" should be set to the version of Picard this plugin to run with.</dl>
+			<dl>"PLUGIN_LICENSE" should be set with the license name of the plugin. You can choose on your desired license <a href="https://spdx.org/licenses/">here</a>.</dl>
+			<dl>"PLUGIN_LICENSE_URL" should be set with the license url.</dl>
+		</ul>
+		<p></p>
+		<p></p>
+		PLUGIN_API_VERSIONS available:
+		<p>v1.0
+			<ul>
+				<dl>0.15.0</dl>
+				<dl>0.15.1</dl>
+				<dl>0.16.0</dl>
+				<dl>1.0.0</dl>
+				<dl>1.1.0</dl>
+				<dl>1.2.0</dl>
+				<dl>1.3.0</dl>
+				<dl>1.4.0</dl>
+			</ul>
+		</p>
+		<p>v2.0
+			<ul>
+				<dl>2.0</dl>
+			</ul>
+		</p>
+		</p>
```

Comment:
This `</p>` should probably be up where you have the two empty `p` elements. You can't have a `p` element inside another `p` element.